# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: C:\Projects\hudson\trunk\hudson\plugins\subversion\src\main\java\hudson\scm\browsers
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: Sventon2.java
--- Sventon2.java Base (BASE)
+++ Sventon2.java Locally Modified (Based On LOCAL)
@@ -78,8 +78,6 @@
      */
     private String getPath(Path path) {
         String s = trimHeadSlash(path.getValue());
-        if(s.startsWith(repositoryInstance)) // this should be always true, but be defensive
-            s = trimHeadSlash(s.substring(repositoryInstance.length()));
         return s;
     }
 
